glsl_builders.py

```diff
@@ -29,6 +29,10 @@ def include_file_in_rd_header(filename, header_data, depth):
 
     while line:
 
+        index = line.find("//")
```

Comment:
This is a very simple comment check. But since this python script ahould only ever be applied to glsl files it should be fine.